labelbox/client.py

```diff
@@ -931,6 +931,30 @@ def delete_unused_feature_schema(self, feature_schema_id):
                 "Failed to delete the feature schema, message: " +
                 response.json()['message'])
 
+    def delete_unused_ontology(self, ontology_id):
+        """
+        Deletes an ontology if it is not used by any annotations
+
+        Args:
+            ontology_id (str): The id of the ontology to delete
+
+        Returns:
+            True if the ontology was deleted
+        """
+
+        endpoint = self.rest_endpoint + "/ontologies/" + ontology_id
```

Comment:
Could you URL encode this and validate `ontology_id` as str of length 25 as well?